apps/main/src/loan/lib/apiCrvusd.ts

```diff
@@ -157,18 +157,24 @@ const detailInfo = {
   loanInfo: async (llamma: Llamma) => {
     log('loanInfo', llamma.collateralSymbol)
     const fetchedPartialLoanInfo = await detailInfo.loanPartialInfo(llamma)
+
+    // Check if we're in beta mode to skip old bands data fetching
+    const isBeta = typeof window !== 'undefined' && localStorage.getItem('release-channel') === 'Beta'
```

Comment:
Window is never undefined anymore, we don't use SSR :partying_face: 
However, please never hardcode the storage keys

See
https://github.com/curvefi/curve-frontend/blob/f8a44864daa3b90f094fe72ee828bd9548d0ab39/packages/curve-ui-kit/src/hooks/useLocalStorage.ts#L38

If needed, just use something like:
https://github.com/curvefi/curve-frontend/blob/f8a44864daa3b90f094fe72ee828bd9548d0ab39/packages/curve-ui-kit/src/hooks/useLocalStorage.ts#L76